packages/shared/src/components/modals/award/GiveAwardModal.tsx

```diff
@@ -345,7 +402,13 @@ const ModalRender = ({ ...props }: ModalProps) => {
         <Modal
           kind={isMobile ? ModalKind.FlexibleTop : Modal.Kind.FlexibleCenter}
           size={Modal.Size.Small}
-          className={classNames(!isMobile ? '!h-[40rem]' : undefined)}
+          className={classNames(
+            !isMobile && activeStep !== 'SUCCESS' ? '!h-[40rem]' : undefined,
```

Comment:
Why is this forced height needed here XD?

Reply:
I don't know, thought you'd know 🙈 
<img width="580" alt="image" src="https://github.com/user-attachments/assets/a65eb114-51c1-4520-9e25-74288a1e1b2b" />

Reply:
Maybe to make the modals more consistent?
![image](https://github.com/user-attachments/assets/5e46fe15-3913-493b-a5f7-350249b8a2c4)

Reply:
Probably because in in the design, the modal has fixed height of 640px

Reply:
Oh right no because they flickered too much with the internal steps.
I see you actually exclude your one :)